packages/node/__tests__/index.test.ts

```diff
@@ -62,12 +61,10 @@ function hydrateCache(lastUpdated) {
 }
 
 declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace
   namespace jest {
     interface Matchers<R> {
-      toHaveDocumentationHeader(res?: ServerResponse): {
-        pass: boolean,
-        message: string
-      }
+      toHaveDocumentationHeader(res?: ServerResponse): R;
```

Comment:
This is the same flow I did when I rewrote `jest-expect-har` in TS: https://github.com/readmeio/jest-expect-har/blob/main/src/index.ts#L4-L14